WORKSPACE

```diff
@@ -73,42 +73,3 @@ envoy_dependencies()
 load("@envoy//bazel:dependency_imports.bzl", "envoy_dependency_imports")
 
 envoy_dependency_imports()
-
-# Docker dependencies
-
-docker_dependencies()
```

Comment:
You can also remove this function in repositories.bzl

Reply:
good catch. Thanks. Fixed.